src/pages/api/citations/[msid]/bibtex.page.ts

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
```

Comment:
Could we use fetch? We deliberately tried to use fetch in other areas to keep client bundle size down, but that doesn't matter here. Still, it feels like it would be more consistent to keep using fetch here instead of importing axios